lighthouse-cli/test/fixtures/ric-shim.html

```diff
@@ -0,0 +1,40 @@
+<html>
```

Comment:
I believe we try to have an HTML copyright header, but it's probably missing from whichever one you copied it from :)

Reply:
yeah copied it from another smoke test html :) i added the header